### PR TITLE
Mulit module change sdk project structure

### PR DESCRIPTION
**Pull request by drEfrati**

### CODE REVIEW

File: kin-recovery/recovery/src/androidTest/java/kin/recovery/qr/QRFileUriHandlerImplTest.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void saveFile_Success() throws Exception {
 		Bitmap bitmap = TestUtils.loadBitmapFromResource(this.getClass(), "qr_test.png");
 		Uri uri = fileUriHandler.saveFile(bitmap);
 		assertThat(uri, notNullValue());
-		assertThat(uri.toString(), equalTo("content://com.kin.ecosystem.backup/qr_codes/backup_qr.png"));
+		assertThat(uri.toString(), equalTo("content://kin.recovery.test.KinRecoveryFileProvider/qr_codes/backup_qr.png"));
```
Comment (yosriz) [inline kin-recovery/recovery/src/androidTest/java/kin/recovery/qr/QRFileUriHandlerImplTest.java]: why is this change? didn't we took care of the URI mismatch?

**Comment by yosriz** [comment]
@drEfrati some checklist of things to validate after this change:
* Jitpack - make sure we didn't change kin-sdk library gradle import, verify it is build correctly
* Testing - I know we have a different task for this, but make sure we didn't change something fundamental and testing are failing because of that.
* Search for relative paths and fix them to the new path (for instance: `readme.md`, `run_integ_test.sh`)

**Comment by drEfrati** [comment]
@yosriz I updated the paths in readme.md and in run_integ_test.sh.
Didn't find anymore places with broken paths.
please look at those commits and confirm.

File: README.md [hunk 2]
```diff
@@ -509,4 +509,4 @@ Now you can build the library using gradle, or open the project using Android St
 
 `Sample` app covers the entire functionality of `kin-sdk`, and serves as a detailed example on how to use the library.
```
Comment (yosriz) [inline README.md]: change "Generated report can be found at" path (see above)

**Pull request merged**